doc/user/structure.rst

```diff
@@ -62,10 +62,17 @@ target
     heterogeneous dishes. The :samp:`{band}` is the single-letter abbreviation
     for the receiver band (``l``, ``s``, ``u`` or ``x``).
 
+    Additionally, for each cohort there is a target
+    :samp:`cohort/{cohort}/{band}`. For MeerKAT the cohort name is simply
+    ``meerkat``. For the MeerKAT Extension it will be ``meerkat_extension``.
```

Comment:
Is `meerkat_extension` the 16 new dishes? I guess so - momentarily confused with MeerKAT+, the whole caboodle.

Reply:
Yes, the new dishes only. I've updated to clarify that.